scalardb-test/src/main/java/kelpie/scalardb/transfer/sql/jdbc/TransferProcessor.java

```diff
@@ -0,0 +1,120 @@
+package kelpie.scalardb.transfer.sql.jdbc;
+
+import com.scalar.db.sql.TransactionMode;
+import com.scalar.kelpie.config.Config;
+import com.scalar.kelpie.modules.TimeBasedProcessor;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.concurrent.ThreadLocalRandom;
+import kelpie.scalardb.transfer.TransferCommon;
+import kelpie.scalardb.transfer.sql.SqlCommon;
+import org.apache.commons.dbcp2.BasicDataSource;
+
+public class TransferProcessor extends TimeBasedProcessor {
+  private final int numAccounts;
+  private final BasicDataSource dataSource;
+
+  public TransferProcessor(Config config) {
+    super(config);
+    numAccounts = (int) config.getUserLong("test_config", "num_accounts");
+    dataSource = SqlCommon.getDataSource(config, TransactionMode.TRANSACTION);
+  }
+
+  @Override
+  public void executeEach() throws Exception {
+    int fromId = ThreadLocalRandom.current().nextInt(numAccounts);
+    int toId = ThreadLocalRandom.current().nextInt(numAccounts);
+    int amount = ThreadLocalRandom.current().nextInt(1000) + 1;
+
+    try (Connection connection = dataSource.getConnection()) {
+      transfer(connection, fromId, toId, amount);
+    }
+  }
+
+  @Override
+  public void close() {
+    try {
+      dataSource.close();
+    } catch (SQLException ignore) {
+      // ignored
+    }
+  }
+
+  private void transfer(Connection connection, int fromId, int toId, int amount)
+      throws SQLException {
+    int fromType = 0;
+    int toType = 0;
+    if (fromId == toId) {
+      toType = 1; // transfer between the same account
+    }
+
+    int fromBalance;
+    int toBalance;
+
+    try (PreparedStatement preparedStatement =
+        connection.prepareStatement(
+            "SELECT * FROM "
```

Comment:
Just a comment. We might be able to have these JDBC-based workloads in `jdbc-test` or something like that for benchmarking vanilla RDBMS if these are purely compatible with JDBC. I guess not the case for the 2PC one, though.

Reply:
That's true. If we need to test to compare between Scalar DB JDBC and a JDBC database in the future, we can do that refactoring so that we can test them with the same code. Thank you.